model/src/guild/auto_moderation/mod.rs

```diff
@@ -0,0 +1,244 @@
+//! Auto Moderation is a feature which allows each guild to set up rules that
+//! trigger based on some criteria. For example, a rule can trigger whenever a
+//! message contains a specific keyword.
```

Comment:
Might be a bit too long 
![image](https://user-images.githubusercontent.com/26655508/175833292-7dee2a6d-261c-433d-a794-981947614a5a.png)